bentoml/_internal/configuration/default_configuration.yaml

```diff
@@ -51,3 +51,8 @@ tracing:
   otlp:
     protocol: Null
     url: Null
+
+logging:
+  formatting:
+    trace_id_format: 032x
+    span_id_format: 016x
```

Comment:
Unfortunately while I think we should have done this to begin with, I don't think we can do this by default as it may break people's setups.

At some point we should work out how to handle default configuration changes, but for now I think we'll have to keep the defaults the way they were before :(

Reply:
Spoke to the team, we're ok with this change :+1: